functests/performance/cpu_management.go

```diff
@@ -76,7 +76,7 @@ var _ = Describe("[performance] CPU Management", func() {
 			Expect(capacityCPU - allocatableCPU).To(Equal(int64(len(listReservedCPU))))
 		})
 
-		It("Verify CPU affinity mask, CPU reservation and CPU isolation on worker node", func() {
+		It("[rfe_id:27363][test_id:28026][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Verify CPU affinity mask, CPU reservation and CPU isolation on worker node", func() {
```

Comment:
How about moving "rfe_id" on line #30 ? 
All tests have one rfe, so we can just add it to main Describe and remove duplicates from